examples/v1/pipelineruns/pipelinerun-with-final-tasks.yaml

```diff
@@ -0,0 +1,203 @@
+# Copied from https://github.com/tektoncd/catalog/blob/v1/git/git-clone.yaml :(
```

Comment:
Link in the comment should be https://github.com/tektoncd/catalog/blob/v1beta1/git/git-clone.yaml - at least until we have a `v1` tag on the catalog